src/extension/kernel.ts

```diff
@@ -141,6 +141,7 @@ export class Kernel implements Disposable {
     this.#experiments.set('escalationButton', config.get<boolean>('escalationButton', false))
     this.#experiments.set('smartEnvStore', config.get<boolean>('smartEnvStore', false))
     this.#experiments.set('aiLogs', config.get<boolean>('aiLogs', false))
+    this.#experiments.set('shellWarning', config.get<boolean>('shellWarning', false))
```

Comment:
I don't see this experiment defined in `package.json`.

Reply:
@sourishkrout I'm aware of that. I think adding the respective entry in the package.json is unnecessary for this kind of temporary flag. But, if you feel it's necessary I'll add it.

Reply:
let's add it. i think it could come handy to enable user to turn it off if they experience problems. more self-explanatory if it's included in `package.json`.

Reply:
@sourishkrout added ✅